utils/compiler.h

```diff
@@ -37,6 +37,19 @@
 #endif
 #endif
 
+#if defined(__riscv)
+#define cpu_relax() asm volatile("nop" ::: "memory")
+#if __riscv_xlen == 64
```

Comment:
Well, when I wrote this code, I didn't know what to do, so I used the `nop` instruction as a temporary fix.

However, it appears that in RISC-V, the `fence` instruction will be used for this part, and this is the same for both `RV32I` and `RV64I`.
<br>

In conclusion, I would combine the 32 and 64 into one and implement the code to use the `fence` instruction.

Reply:
I was going to implement it with the `fence` instruction, but then I realized that RISC-V has an additional `fence.i` instruction for memory barriers.

So when I implement a function that calls this macro function, I want to implement that macro function as well.

Do you think this would be okay?